File: NativeTestSample/NativeTestSample/unittest1.cpp
#include "stdafx.h"
#include "CppUnitTest.h"

using namespace Microsoft::VisualStudio::CppUnitTestFramework;

namespace NativeTestSample
{		
	TEST_CLASS(UnitTest1)
	{
	public:
		
		TEST_METHOD(TestMethod1)
		{
			// TODO: Your test code here
		}

		TEST_METHOD(TestAssertVariants)
		{
			Assert::AreEqual((long long)2, (long long)2, L"works!");
		}

	};
}

File: TestWithBindingRedirect/TestWithBindingRedirect/UnitTest1.cs
﻿using System;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace TestWithBindingRedirect
{
    [TestClass]
    public class UnitTest1
    {
        [TestMethod]
        public void TestMethod1()
        {
            new ClassLibrary1.Class1().Foo();
        }
    }
}
